docs/Diagnostics.md

```diff
@@ -72,4 +72,52 @@ First off, you're going to need xperf and wpa. Installing the [Windows ADK](http
 
 ## Counters
 
+To assist investigations into running systems, MsQuic has a number of performance counters that are updated during runtime. These counters are exposed as an array of unsigned 64-bit integers, via a global `GetParam` parameter.
+Sample code demonstrating how to query the performance counters:
+```c
+uint64_t Counters[QUIC_PERF_COUNTER_MAX];
+uint32_t BufferLength = sizeof(Counters);
+MsQuic->GetParam(
+    NULL,
+    QUIC_PARAM_LEVEL_GLOBAL,
+    QUIC_PARAM_GLOBAL_PERF_COUNTERS,
+    &BufferLength,
+    Counters);
+```
+
+Each of the counters available is described here:
+Counter | Description
+--------|------------
+QUIC_PERF_COUNTER_CONN_CREATED | Total connections ever allocated.
```

Comment:
Some of these have periods at the end, and some don't. Let's just remove all the periods.